### PR TITLE
MSPSDS-526: update case page to tabs layout

### PR DESCRIPTION
**Pull request by jasiekmiko**
An extension to the MSPSDS-526 work, klling the investigation_presenter.
This allows all of our code to view logic to remain within the views

### CODE REVIEW

File: web/test/controllers/investigations_controller_test.rb [hunk 1]
```diff
@@ -42,6 +42,11 @@ class InvestigationsControllerTest < ActionDispatch::IntegrationTest
     assert_response :success
   end
 
+  test "should generate investigation pdf" do
+    get investigation_url(@investigation) + ".pdf"
```
Comment (broder) [inline web/test/controllers/investigations_controller_test.rb]: Does `investigation_url(@investigation, format: :pdf)` work?

File: web/app/presenters/tab.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 class Tab
-  attr_reader :id, :title, :partial
+  attr_reader :id, :title, :render
 
   def initialize(id, title, partial)
     @id = id
     @title = title
-    @partial = partial
+    @render = partial
```
Comment (broder) [inline web/app/presenters/tab.rb]: Maybe if you're changing the below, you could change the argument to be `render` rather than `partial`.

Comment (jasiekmiko) [reply web/app/presenters/tab.rb]: ah good catch thanks!

File: web/test/controllers/investigations_controller_test.rb (hunk 1 above)
Comment (jasiekmiko) [reply web/test/controllers/investigations_controller_test.rb]: yeah! thanks! :)

**Pull request merged**